documentation/modules/ROOT/pages/transformations/timezone-converter.adoc

```diff
@@ -0,0 +1,210 @@
+----
+
+=== Effect of applying the `TimezoneConverter` SMT
+The following examples show the timestamp fields in an event record before and after applying the `TimezoneConverter` transformation.
+
+.Value of `created_at` field in an event record before processing by the `TimezoneConverter` transformation
+====
+The value of the `created_at` field shows the UTC time.
+
+[source, json]
+----
+{
+        "before": null,
+        "after": {
+            "id": 1,
+            "first_name": "Anne",
+            "last_name": "Kretchmar",
+            "email": "annek@noanswer.org",
+            "created_at": "2011-01-11T16:40:30.123456789+00:00"
+        },
+        "source": {
+            "version": "2.4.0.Aplha2",
+            "connector": "postgresql",
+            "name": "PostgreSQL_server",
+            "ts_ms": 1559033904863,
+            "snapshot": true,
+            "db": "postgres",
+            "sequence": "[\"24023119\",\"24023128\"]",
+            "schema": "public",
+            "table": "customers",
+            "txId": 555,
+            "lsn": 24023128,
+            "xmin": null
+        },
+        "op": "c",
+        "ts_ms": 1559033904863
+    }
+----
+
+====
+.Event record value after processing by the `TimezoneConverter` transformation
+====
+The SMT converts the original UTC value of the `created_at` field to the time in the target `Pacific/Easter` timezone that is specified in the xref:basic-example-timezone-converter[Basic configuration] example.
+The SMT also adds an `event_timestamp` field.
+
+[source, json]
+----
+{
+        "before": null,
+        "after": {
+            "id": 1,
+            "first_name": "Anne",
+            "last_name": "Kretchmar",
+            "email": "annek@noanswer.org",
+            "created_at": "2011-01-11T11:40:30.123456789-05:00"
+        },
+        "source": {
+            "version": "2.4.0.Aplha2",
+            "connector": "postgresql",
+            "name": "PostgreSQL_server",
+            "ts_ms": 1559033904863,
+            "snapshot": true,
+            "db": "postgres",
+            "sequence": "[\"24023119\",\"24023128\"]",
+            "schema": "public",
+            "table": "customers",
+            "txId": 555,
+            "lsn": 24023128,
+            "xmin": null,
+            "id": 100
+        },
+        "op": "c",
+        "ts_ms": 1559033904863,
+        "event_timestamp": 1626102708861
+    }
+----
+====
+
+[[advanced-example-timezone-converter]]
+== Example: Advanced configuration
+
+Instead of converting all  timestamp fields in an event record, you can configure the SMT to convert specific fields only.
+The following example shows how you might use the `include.fields` option in the SMT configuration to add convert only the `created_at`, and `updated_at` timestamp fields in an event record.
+Instead of specifying a geographic timezone designator,  this configuration uses a fixed offset to convert the  time from UTC to `+05:30`.
```

Comment:
```suggestion
Instead of specifying a geographic timezone designator, this configuration uses a fixed offset to convert the time from UTC to `+05:30`.
```